### PR TITLE
Add cucushift-logging-e2e step for Logging

### PR DESCRIPTION
**Pull request by IshwarKanse**
This is to fix the hard-coded serial label when running parallel_cucumber and allows us to customize the Logging tests execution further.

### CODE REVIEW

File: ci-operator/step-registry/cucushift/logging-e2e/cucushift-logging-e2e-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      ci-operator.openshift.io/shm: 2G
+      cpu: 800m
+      memory: 1Gi
+  env:
+  - name: E2E_RUN_TAGS
+    default: "@aws-ipi"
+    documentation: "Executed cucumber tags"
+  - name: E2E_SKIP_TAGS
+    default: "not @customer
+          and not @destructive
```
Comment (QiaolingTang) [inline ci-operator/step-registry/cucushift/logging-e2e/cucushift-logging-e2e-ref.yaml:26]: Most of the logging cases have `@destructive` tag, so I think we should not add this tag here.

File: ci-operator/step-registry/cucushift/logging-e2e/cucushift-logging-e2e-commands.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+show_time_used "$timestamp_start" 'normal'
+
+# run admin tests
+export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/parallel/admin"
+timestamp_start="$(date +%s)"
+parallel_cucumber -n "${PARALLEL}" ${PARALLEL_CUCUMBER_OPTIONS} --exec \
+    'export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS=$(echo ${USERS} | cut -d "," -f ${TEST_ENV_NUMBER},$((${TEST_ENV_NUMBER}+${PARALLEL})),$((${TEST_ENV_NUMBER}+${PARALLEL}*2)),$((${TEST_ENV_NUMBER}+${PARALLEL}*3)));
+     export WORKSPACE=/tmp/dir${TEST_ENV_NUMBER};
+     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and ${E2E_SKIP_TAGS} and not @console and @admin\" -p junit"' || true
+show_time_used "$timestamp_start" 'admin'
```
Comment (QiaolingTang) [inline ci-operator/step-registry/cucushift/logging-e2e/cucushift-logging-e2e-commands.sh:61]: If I'm not wrong, logging cases in cucushift can only be tested  in serial, so I think these 2 steps are not needed.

Comment (anpingli) [reply ci-operator/step-registry/cucushift/logging-e2e/cucushift-logging-e2e-commands.sh:61]: Acked

File: ci-operator/step-registry/cucushift/logging-e2e/cucushift-logging-e2e-commands.sh [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+# run the rest tests in serial
+export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/serial"
+export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS="${USERS}"
```
Comment (QiaolingTang) [inline ci-operator/step-registry/cucushift/logging-e2e/cucushift-logging-e2e-commands.sh:65]: @IshwarKanse could you please also set below env var? The `channel` should be the exact channel the job will test. 
```
export BUSHSLICER_CONFIG='
  environments:
    ocp4:
      logging_envs:
        clo:
          catsrc: "qe-app-registry"
          channel: "stable"
        eo:
          catsrc: "qe-app-registry"
          channel: "stable"
'
```

File: ci-operator/step-registry/cucushift/logging-e2e/cucushift-logging-e2e-commands.sh (hunk 2 above)
Comment (anpingli) [reply ci-operator/step-registry/cucushift/logging-e2e/cucushift-logging-e2e-commands.sh:61]: Logging case should be test  in serial,

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-openshift-logging-5.7-aws-ipi-ovn-ipsec-p1-f7-openshift-logging-5-7-disruptive periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-gcp-ipi-ovn-ipsec-p1-f7-openshift-logging-5-7

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-openshift-logging-5.5.yaml [hunk 4]
```diff
@@ -71,7 +71,7 @@ tests:
       SIZE_VARIANT: large
       TAG_VERSION: '@logging5.5'
       TEST_ADDITIONAL: ""
-      TEST_FILTERS: CPaasrunOnly;CPaasrunBoth
+      TEST_FILTERS: ~Disruptive&;~disruptive&
```
Comment (QiaolingTang) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-openshift-logging-5.5.yaml:74]:
```suggestion
      TEST_FILTERS: CPaasrunOnly;CPaasrunBoth;~Disruptive&;~disruptive&
```
I think we still need `CPaasrunOnly;CPaasrunBoth;`

Comment (IshwarKanse) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-openshift-logging-5.5.yaml:74]: It shouldn't make any difference if we specify the tag or not. All of our cases have either of the tags.

Comment (QiaolingTang) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-openshift-logging-5.5.yaml:74]: No, this case https://github.com/openshift/openshift-tests-private/blob/master/test/extended/logging/acceptance.go#L47 doesn't have these tags, and it's designed for OCP testing only.

**Pull request closed without merging**